### PR TITLE
sw/interface: Add 0258-beta3 DAPLink firmware for V2.00 & V2.2*.

### PR DESCRIPTION
**Pull request by microbit-carlos**
In preparation to share the DAPLink beta with beta testers.

### CODE REVIEW

File: software/interface.md [hunk 1]
```diff
@@ -75,20 +75,30 @@ This table shows the latest DAPLink release for each board version that has been
 
 ### Beta releases
 
-#### DAPLink 0258-beta1
+#### DAPLink 0258-beta3
 
-This DAPLink beta release has been built with the GCC toolchain and targets the micro:bit V2.00 board. It also includes stability improvements for data logging.
+This DAPLink beta release for V2.00 and V2.2* has the following improvements:
```
Comment (microbit-robert) [inline software/interface.md:80]: Consider use of the tags we use elsewhere for version for consistency? E.g., `<span class="v2">V2.2x</span>`.

**Review by microbit-robert — APPROVED** [review]
This looks good to me. One suggestion you can take or leave. Happy for you to merge either way.

File: software/interface.md (hunk 1 above)
Comment (microbit-carlos) [reply software/interface.md:80]: Good call, I'll update and push again 👍

Comment (microbit-carlos) [reply software/interface.md]: Updated in https://github.com/microbit-foundation/dev-docs/compare/b1caaf7062e3a6a2105fa79afa4c76e8a48f704f..8ba0f340bef8ea9fb38c43f0d031aa6c1366cef1

**Comment by microbit-carlos** [comment]
Thanks Rob!

**Pull request merged**